index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">

```

Comment:
The main content of your page is NOT in english.

Reply:
Corrected!